Makefile

```diff
@@ -9,14 +9,6 @@ endif
 PROJECT := github.com/juju/juju
 PROJECT_DIR := $(shell go list -e -f '{{.Dir}}' $(PROJECT))
 
-ifeq ($(shell uname -p | sed -r 's/.*(86|armel|armhf|aarch64|ppc64le|s390x).*/golang/'), golang)
-	GO_C := golang-1.8
-	INSTALL_FLAGS :=
-else
-	GO_C := gccgo-4.9  gccgo-go
-	INSTALL_FLAGS := -gccgoflags=-static-libgo
```

Comment:
`INSTALL_FLAGS` is still referenced elsewhere. I think only `GO_C` can be removed.